src/components/cc-tile-deployments/cc-tile-deployments.js

```diff
@@ -63,49 +55,49 @@ export class CcTileDeployments extends LitElement {
   }
 
   render () {
+    if (this.state.type === 'error') {
+      return html`
+        <div class="tile_message">
+          <div class="error-message">
+            <cc-icon .icon="${iconAlert}" a11y-name="${i18n('cc-tile-deployments.error.icon-a11y-name')}" class="icon-warning"></cc-icon>
+            <p>${i18n('cc-tile-deployments.error')}</p>
+          </div>
+        </div>
+      `;
+    }
```

Comment:
**❔ question:** Maybe intended but the `error` state does not show the title anymore.

Reply:
Obviously a regression. I'll fix that. Tank you